src/main/java/by/custom/utilcalculator/service/UserProgressManager.java

```diff
@@ -33,93 +35,123 @@ public void createNewUserProgress(final String chatID) throws UtilsborException
     }
 
     public String processCarOrigin(final String requestingCommand, final String chatID) throws UtilsborException {
-        UserProgress userProgress;
+        final UserProgress userProgress;
         userProgress = userProgressStorage.get(chatID);
-        String message;
-        switch (requestingCommand) {
-            case Command.EAES -> userProgress.setCountryOrigin(CountryOrigin.EAES);
-            case Command.OTHER_COUNTRIES -> userProgress.setCountryOrigin(CountryOrigin.OTHER);
-
+        final String message;
+
+        if (Objects.equals(requestingCommand, Command.EAES.getCommand())) {
+            userProgress.setCountryOrigin(CountryOrigin.EAES);
+        } else {
```

Comment:
```suggestion
        } else if (Objects.equals(requestingCommand, Command.OTHER_COUNTRIES.getCommand())) {
            userProgress.setCountryOrigin(CountryOrigin.OTHER);
        }
```

Reply:
We have an agreement always use {} after if and else, even if it is not necessary.
I removed {} but probably we need to discuss it during our call